lib/tc.js

```diff
@@ -4,7 +4,7 @@ const sudo = require('./sudo');
 
 async function getDefaultInterface() {
   const result = await shell(
-    "sudo route | grep '^default' | grep -o '[^ ]*$' | tr -d '\n'",
+    "sudo route | grep -m 1 '^default' | grep -o '[^ ]*$' | tr -d '\n'",
```

Comment:
Double checked on my macbook (`grep (BSD grep) 2.5.1-FreeBSD`) and this command looks to be supported

Reply:
That should be ok since we do not run that code path on a Mac!